packages/sdks/web-component/src/lib/helpers/helpers.ts

```diff
@@ -504,6 +534,7 @@ export const showFirstScreenOnExecutionInit = (
     oidcLoginHint,
     oidcPrompt,
     oidcErrorRedirectUri,
+    scopes,
```

Comment:
Don't we need to send the state as well?

Reply:
fixed